### DISCUSSION TITLE
P6000 Documentation

### DISCUSSION THREAD

**Discussion by CompPilot** [image-et-son/p600fw#113] [General]
Hi Image-et-son/imogen,

Thanks for all your hard work on this project! I saw your request for proofreading of the P6000* documentation on GS so I signed up here on GitHub to help. I've written a few manuals over the years.

*(P6000 is how I like to refer to the P600 with all the enhancements to date, arguably improved in capability by an order of magnitude.)

First, comment: Your new manual is very impressive! Thank you for putting it together. 

If I were to suggest one overall change it would be to go with a more personal active voice rather than the dry and impersonal passive voice. That would make it more engaging and readable. 

To start with, here are a couple of typos I noticed, on page 49:
"2. Unlike the internal key on/off events, in Shift Mode or Shift Lock Mode incoming MIDI notes do no cause transpositions but a re ignored"

Probably meant to be "2. Unlike the internal key on/off events, in Shift Mode or Shift Lock Mode incoming MIDI notes do not cause transpositions but are ignored"

Even better would be: 
2. When in Shift Mode or Shift Lock Mode, incoming MIDI notes are ignored and do not cause transpositions. Internal key on/off events will still cause transpositions.

Active voice option: 
When you select Shift Mode or Shift Lock Mode, the P6000 will simply ignore MIDI notes from external keyboards or sequencers and they will not cause transpositions. You can still play notes on the P6000's own keyboard to create transpositions.

That's just something that caught my eye as I was glancing through it. I'll go over it more over the next few weeks as time permits.

Thanks,
 -CompPilot

**Comment by image-et-son** [comment]
Hi there, yes I agree. I will sift through it to change it to personal active. Thanks for the feedback. Unfortunately, this post from you totally slipped my attention. I did not get a notification for it, so sorry for the late reply. 

Any further feedback would be very welcome!
